config/data_node.go

```diff
@@ -26,6 +28,8 @@ func (manager *ConfigurationManager) DataNodeConfig() *DataNodeconfig {
 			NameNodeIP:                   envString("NAME_NODE_IP", "127.0.0.1"),
 			InternalRequestsPort:         envString("INTERNAL_REQ_PORT", "6000"),
 			NameNodeInternalRequestsPort: envString("NAME_NODE_INTERNAL_REQ_PORT", "7000"),
+			UploadEndpoint:               envString("UPLOAD_ENDPOINT", "/upload"),
```

Comment:
I think we either leave this in the code or have an endpoints.yaml config that defines them. I say leave it in the code, but it's not an env var usecase.